pyproject.toml

```diff
@@ -49,6 +49,7 @@ cwltool = "==3.1.20211107152837"
 dictobj = "*"
 docutils = "!=0.17, !=0.17.1"
 edam-ontology = "*"
+email-validator = "*"
```

Comment:
I think it'd be more future-proof to replace this:

```suggestion
```

by modifying the pydantic requirement below to:

```
pydantic = {version = "*", extras = ["email"]}
```

as suggested in the import error raised in the API tests: ImportError: email-validator is not installed, run `pip install pydantic[email]`